lib/eventsource.js

```diff
@@ -34,10 +34,10 @@ function EventSource (url, eventSourceInitDict) {
   var self = this
   self.reconnectInterval = 1000
 
-  function onConnectionClosed () {
+  function onConnectionClosed (message=undefined) {
```

Comment:
If you don't provide `message` argument, it's already `undefined` by default, so it's superfluous. Please remove the default value.

Reply:
Whoops, TS linter artifact I forgot to cleanup, thanks!

Reply:
You are welcome :-)